### PR TITLE
Remove default architecture files and force user to manually setup target device

### PR DESCRIPTION
**Pull request by KochynVolodymyr**
### Motivate of the pull request
Force users to setup target device or architecture files.

### Describe the technical details
Remove default architecture files.

### Impact of the pull request
All scripts should manually defines target device or architecture files.

### CODE REVIEW

File: tests/Testcases/trivial/test.tcl [hunk 1]
```diff
@@ -1,4 +1,5 @@
 create_design test
+architecture ../../Arch/k6_frac_N10_tileable_40nm.xml ../../Arch/k6_N10_40nm_openfpga.xml
```
Comment (ravic-rs) [inline tests/Testcases/trivial/test.tcl:2]: Can we add a negative test in one of the top-level cases ?

Comment (KochynVolodymyr) [reply tests/Testcases/trivial/test.tcl:2]: No, we can't. let's discuss this case later.

File: tests/Testcases/bgm/bgm.tcl [hunk 2]
```diff
@@ -1,4 +1,5 @@
 create_design BGM
+architecture ../../Arch/k6_frac_N10_tileable_40nm.xml ../../Arch/k6_N10_40nm_openfpga.xml
```
Comment (ravic-rs) [inline tests/Testcases/bgm/bgm.tcl:2]: Can provide fullpaths here for readability/search?

Comment (ravic-rs) [reply tests/Testcases/bgm/bgm.tcl:2]: Compiler expects the files to be in path.

Comment (ravic-rs) [reply tests/Testcases/bgm/bgm.tcl:2]: Aids with moving the files without affecting the script.

File: tests/Testcases/trivial/test.tcl (hunk 1 above)
Comment (ravic-rs) [reply tests/Testcases/trivial/test.tcl:2]: Dummy compiler

Comment (ravic-rs) [reply tests/Testcases/trivial/test.tcl:2]: Test will be added at Raptor level.

**Review by ravic-rs — COMMENTED** [review]
Needs clarification around the order of operations in tcl script,
design file and set device name etc.

**Comment by ravic-rs** [comment]
> Needs clarification around the order of operations in tcl script, design file and set device name etc.

create_design is required before target device or architecture file selection

**Review by ravic-rs — COMMENTED** [review]
On hold for checks upstream.

**Review by ravic-rs — APPROVED** [review]
Looks good.

**Pull request merged**